programs/zstdcli.c

```diff
@@ -691,6 +693,7 @@ int main(int const argCount, const char* argv[])
                 if (!strcmp(argument, "--rm")) { FIO_setRemoveSrcFile(prefs, 1); continue; }
                 if (!strcmp(argument, "--priority=rt")) { setRealTimePrio = 1; continue; }
                 if (!strcmp(argument, "--output-dir-flat")) {nextArgumentIsOutDirName=1; lastCommand=1; continue; }
+                if (!strcmp(argument, "--diff-from")) {nextArgumentIsDiffFromDictFileName=1; lastCommand=1; continue; }
```

Comment:
consider using `--diff-from=#` instead.

Advantages are : 
- Reading the file name is done in the same operation. This avoids to "split" the logic in 2 different places and times, adding more code, and with potentially something going wrong in between.
- This policy is more consistent with other CLI arguments. I know we do have a few old arguments left that still use the 2-reads strategy, but this is now discouraged, and may eventually be displaced if possible (as an example, I really believe that the recent `--output-dir-flat` should be removed, and only `--output-dir-flat=#` should be advertised and remain (this is a different topic, unrelated to this PR)).

Reply:
Ah sure. I was wondering why there were two instances of the same command. I have --diff-from=# in there. I'll just go ahead and remove this endpoint then.